elastic/kibana
# [ResponseOps][Alerting] create new logger with tag for rule/connector type, for logger given to executors

Resolves https://github.com/elastic/kibana/issues/115099
## Summary

Made changes to provide loggers to rule and connector executors. Instead of giving them the logger we currently give them, I gave them a new one that has an additional tag in it, indicating the rule/connector type id.

### Checklist


### To verify

- Create stack rule with a connector
- Let it run, and verify that the logger tags on rule run also include the rule and connector id.
